src/pke/include/constants-fwd.h

```diff
@@ -0,0 +1,136 @@
+//==================================================================================
```

Comment:
I am aware about this situation, but didn't have time to fix it. it will be resolved differently, without creating a new file. just a few line code change